example/src/App.tsx

```diff
@@ -430,7 +442,6 @@ const styles = StyleSheet.create({
     backgroundColor: '#000000',
     position: 'absolute',
     width: '100%',
-    height: AppLovinMAX.getAdaptiveBannerHeightForWidth(-1),
```

Comment:
What's the height now?

Reply:
AdView automatically sizes the height based on the current AdView type and the device type.  

How about allowing to set `auto`?   Since this is out of the main function, only the static functions can be used, but `getAdaptiveBannerHeightForWidth` is no longer a static function.

I'll update the height to use `auto`.  Please review.

Reply:
Yeah, allowing to set to `auto` makes sense - up to you if you want to keep it in this same PR (we usually untangle changes into different PRs).

Reply:
Let me update this in the same PR since `getAdaptiveBannerHeightForWidth` has to be removed.